internal/resolution/client/osv_offline_client.go

```diff
@@ -0,0 +1,94 @@
+package client
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"sync"
+
+	"deps.dev/util/resolve"
+	"github.com/google/osv-scanner/internal/local"
+	"github.com/google/osv-scanner/internal/resolution/util"
+	"github.com/google/osv-scanner/pkg/models"
+	"github.com/google/osv-scanner/pkg/osv"
+)
+
+type OSVOfflineClient struct {
+	// TODO: OSV-Scanner v2 plans to make vulnerability clients that can be used here.
+	offline     bool
```

Comment:
`offline` is a bit confusing here - sounds like this indicates if we query the local database for vulnerabilities, but this is already an offline client. any reason that we don't use `downloadDBs` here?

Reply:
Ended up removing this variable.